source/extensions/extensions_metadata.yaml

```diff
@@ -664,6 +664,11 @@ envoy.wasm.runtime.v8:
   - envoy.wasm.runtime
   security_posture: unknown
   status: alpha
+envoy.wasm.runtime.wamr:
+  categories:
+  - envoy.wasm.runtime
+  security_posture: unknown
+  status: alpha
```

Comment:
@leyao-daily can you put move thte build comments I previously asked for to YAML comments here? Thanks.
/wait

Reply:
Got it. And same for wasmtime and wavm?